### DISCUSSION TITLE
timezone setting and checking, and 'demand' slider

### DISCUSSION THREAD

**Discussion by antwin** [revk/xESP32-Faikout#437] [Q&A] [answered]
I have set the timezone to NZST-12NZDT,M9.5.0,M4.1.0/3
which - by analogy - seems to be the setting for New Zealand. 
However, state/faikin/  starts with {"ts":"2024-08-14T09:07:00Z",
which looks like GMT to me. How do I actually know what time the faikin really thinks it is, in case I got the setting wrong? Could we have it on the web display, for reassurance?
And what does the 'demand' slider do?

**Comment by revk** [comment, marked as answer]
`"ts":"2024-08-14T09:07:00Z"` is the timestamp on an MQTT message and is reported in ISO8601 format, the `Z` suffix means it is reporting UTC.

However, for any operations, such as times start/stop, etc, it will be working in the time zone you have set.

If you go to *Settings* and select *Basic* the bottom of the page should have details, including current time.

E.g. my units say

<img width="282" alt="Screenshot 2024-08-14 at 10 18 03" src="https://github.com/user-attachments/assets/094f6c65-2224-49f8-9d42-04437f3c0b7a">
